### DISCUSSION TITLE
Welcome to pypackage-template Discussions!

### DISCUSSION THREAD

**Discussion by browniebroke** [browniebroke/pypackage-template#173] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by abe-101** [comment]
Hi everyone! :wave: 

I'm Abe, a full stack developer specializing in building custom business software using Django.

I recently used this template for my Python module called [python-mergecal](https://github.com/mergecal/python-mergecal), which merges calendars. This module powers my hobby project, [MergeCal.org](https://mergecal.org).

I'm excited to be part of this community and look forward to sharing ideas and learning from others. Feel free to check out my project and let me know what you think!

**Reply by browniebroke** [reply]
Hello and welcome!
